rllib/offline/json_writer.py

```diff
@@ -113,9 +113,32 @@ def _to_jsonable(v, compress: bool) -> Any:
         return str(pack(v))
     elif isinstance(v, np.ndarray):
         return v.tolist()
+
     return v
 
 
+def _convert_keys_to_strings_json_dict(d: Dict[Any, Any]) -> Dict[Any, Any]:
```

Comment:
I am actually not sure if you can do this.
like what should we do after we read this json data back in? how would we know which key to convert back into integers?
maybe just clear infos column here for now?

Reply:
Ok I'm in agreement sure.